docs/sphinx/cookbook/dynamic_pool_heuristics.rst

```diff
@@ -1,11 +1,11 @@
 .. _dynamic_pool_heuristics:
 
 ===========================================================
-Improving DynamicPool Performance with a Coalesce Heuristic
+Improving DynamicPoolList Performance with a Coalesce Heuristic
```

Comment:
Is there already a `QuickPool` document for this section?  If not, I suggest that we change this to be more generic.  Somthing like:
```suggestion
Improving Pool Performance with a Coalesce Heuristic
```
We could consider renaming this entire cookbook section and provide code examples for both QuickPool and DynamicPoolList.  We could also talk about why some heuristics are better for QuickPool than for DynamicPoolList.  I would be fine with not changing this now if we could get it documented in a Jira ticket.

Reply:
I think that's a great idea because i do think this needs some updating that's probably best for a separate PR. (Especially the discussion of different heuristics.) I can make a JIRA ticket!

Reply:
See UM-905